windows_scripts/Generate Beatmap.bat

```diff
@@ -0,0 +1,31 @@
+:main
+    @echo off
+	
+	for %%d in (%~dp0..) do set ParentDirectory=%%~fd
+	cd "%ParentDirectory%"
+	
+	SET /P USE_TIMING_POINTS="Use timing points from a beatmap (Y/[N])?: "
+
+	set /p ModelCheckpointPath="Model Checkpoint Path: "
+	set /p SongPath="Song Path: "
+	IF /I "%USE_TIMING_POINTS%" == "Y" set /p TimingPointsBeatmapPath="Beatmap path to take timing points from: "
+	set /p Artist="Artist: "
+	set /p Title="Title: "
+	IF /I not "%USE_TIMING_POINTS%" == "Y" set /p BPM="BPM: "

```

Comment:
would it be possible for you to implement optional bpm?

Reply:
Input 0 to skip?